mesonbuild/interpreter.py

```diff
@@ -383,23 +387,27 @@ def __init__(self, cross_info):
                 'Machine info is currently {}\n'.format(cross_info) +
                 'but is missing {}.'.format(minimum_cross_info - set(cross_info)))
         self.info = cross_info
-        self.methods.update({'system': self.system_method,
-                             'cpu': self.cpu_method,
-                             'cpu_family': self.cpu_family_method,
-                             'endian': self.endian_method,
-                             })
+        self.held_object = environment.MachineInfo(cross_info['system'],
+                                                   cross_info['cpu_family'],
+                                                   cross_info['cpu'],
+                                                   cross_info['endian'])
+        self.methods.update({'system' : self.system_method,
+                             'cpu' : self.cpu_method,
+                             'cpu_family' : self.cpu_family_method,
+                             'endian' : self.endian_method,
+                            })
```

Comment:
__[flake8]__
* [E124] closing bracket does not match visual indentation

<!-- Comment by [SideCI](https://sideci.com) -->